src/main/java/org/apache/mesos/hdfs/state/PersistentState.java

```diff
@@ -71,6 +72,22 @@ public void setFrameworkId(FrameworkID frameworkId) throws InterruptedException,
     zkState.store(value).get();
   }
 
+  public void resetDeadNodeTimeStamps() {
+    Date date = DateUtils.addSeconds(new Date(), conf.getDeadNodeTimeout());
+
+    if (getDeadJournalNodes().size() > 0) {
+      deadJournalNodeTimeStamp = new Timestamp(date.getTime());
```

Comment:
Looking at the previous PR's discussion/outcome, I still think that it's worth creating a timestamp per dead JN/NN.
"I will still argue that the "proper" solution would be a timeout per host, per node-type, but what you have now is good enough to merge in as is, as long as we can track my proposed solution in a TODO/issue somewhere."
Can we please create a TODO/issue to track this? It won't hold up this PR, but still needs to be tracked somewhere.